XpressAI/xircuits
# ✨ Add ExceptionHandler component

# Description

This PR adds `ExceptionHandler` component. Useful for workflows that might hit an exception. This component allows users to handle it gracefully. Attach the main execution to its body branch, and if it hits an exception it will continue on the exception branch, with the exception message. It will then continue with the main execution whether it hits the exception or not.

![image](https://github.com/user-attachments/assets/2307b294-2a9a-4800-897e-d8d71ecf5159)

## Pull Request Type


## Type of Change


# Tests

1. Run the ExceptionHandlerExample.xircuits workflow. Ensure it executes as expected.
2. run any other workflows that may run into exceptions. Ensure that the ExceptionHandler component can handle it as expected.

## Tested on?
